plugins/modules/swim_workflow_manager.py

```diff
@@ -177,21 +177,21 @@
             description: SWIM image name which will be tagged or untagged as golden.
             type: str
           device_role:
-            description: Defines the device role, with permissible values including ALL, UNKNOWN, ACCESS, BORDER ROUTER,
-              DISTRIBUTION, and CORE.
-              ALL - This role typically represents all devices within the network, regardless of their specific roles or functions.
-              UNKNOWN - This role is assigned to devices whose roles or functions have not been identified or classified within Cisco Catalsyt Center.
-                This could happen if the platform is unable to determine the device's role based on available information.
-              ACCESS - This role typically represents switches or access points that serve as access points for end-user devices to connect to the network.
-                These devices are often located at the edge of the network and provide connectivity to end-user devices.
-              BORDER ROUTER - These are devices that connect different network domains or segments together. They often serve as
-                gateways between different networks, such as connecting an enterprise network to the internet or connecting
-                multiple branch offices.
-              DISTRIBUTION - This role represents function as distribution switches or routers in hierarchical network designs. They aggregate traffic
-                from access switches and route it toward the core of the network or toward other distribution switches.
-              CORE - This role typically represents high-capacity switches or routers that form the backbone of the network. They handle large volumes
-                of traffic and provide connectivity between different parts of network, such as connecting distribution switches or
-                providing interconnection between different network segments.
+            description: |
+              Specifies the device role(s) for tagging or untagging the image as golden.
+              Permissible values:
+              - `ALL`: Applies the golden tag to all devices, regardless of role.
+              - `UNKNOWN`: For devices without a specific classification.
+              - `ACCESS`: Devices connecting end-user devices (e.g., access switches).
+              - `BORDER ROUTER`: Devices linking different network segments or domains.
+              - `DISTRIBUTION`: Devices aggregating traffic towards the core.
```

Comment:
- 'DISTRIBUTION': Tags devices aggregating traffic toward the core.

Reply:
updated the code